_includes/header.html

```diff
@@ -61,15 +62,16 @@
 						</hgroup>
 
 						<div class="download">
-	  						<a href="{{ site.tgmpa.github }}/archive/{{ site.tgmpa.version }}.zip" class="latest-zip">
-								<img border="0" width="90" alt="Download latest release zipfile" src="{{ site.tgmpa.zipimage }}">
-							</a>
-	  						<a href="{{ site.tgmpa.github }}/archive/{{ site.tgmpa.version }}.tar.gz" class="latest-tar">
-								<img border="0" width="90" alt="Download latest release tarball" src="{{ site.tgmpa.tarimage }}">
-							</a>
+							<p class="download-icon">
+								<a href="{{ '/download/' | prepend: site.tgmpa.url }}" title="Download the latest release">
```

Comment:
If you've got it in plain text (good), remove it the title attribute. It serves no purpose. For those who have got title attributes being read out in screen readers (not the default), it would read the same thing out twice.

Really, this is all a done deal in the accessibility expertise. Title attributes should not be used except on `iframe` elements.